src/components/AppShelf/AppShelfItem.tsx

```diff
@@ -1,21 +1,21 @@
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
 import {
-  Text,
-  Card,
   Anchor,
   AspectRatio,
+  Card,
   Center,
   createStyles,
-  useMantineColorScheme,
   Image,
+  Text,
+  useMantineColorScheme,
 } from '@mantine/core';
 import { motion } from 'framer-motion';
 import { useState } from 'react';
-import { useSortable } from '@dnd-kit/sortable';
-import { CSS } from '@dnd-kit/utilities';
-import { serviceItem } from '../../tools/types';
 import PingComponent from '../../modules/ping/PingModule';
-import AppShelfMenu from './AppShelfMenu';
 import { useConfig } from '../../tools/state';
+import { serviceItem } from '../../tools/types';
```

Comment:
Your IDE sorts imports alphabetically?

Reply:
Hahaha I guess so! Must be something in my Prettier config